docs/annexes/spdx-lite.md

```diff
@@ -121,7 +121,7 @@ Additionally:
 - Mandatory
     1. created
     1. createdBy (may be multiple), SHOULD be objects of type /Core/Agent
-    1. specVersion, MUST be a fixed string, “3.0.1”.
+    1. specVersion, MUST be a fixed string, “3.0.*” - where * is any supported dot version of the SPDX specification.
```

Comment:
```suggestion
    1. specVersion, MUST be a fixed string, “3.0.*” - where * is any supported dot version of the SPDX specification
```
Stylistically, other list items in this page do not have fullstops. (The fullstop is not introduced by this PR, it was there before) But I will leave this as optional so it will not blocking.